### PR TITLE
Add switzerland as regional communities

### PR DESCRIPTION
**Pull request by lldelisle**
Hi,
We would like to add switzerland as a regional community. @natefoo would you mind to create the mailing list if our community is accepted?

Lucille on behalf of [Hans-Rudolf](@hrhotz) and myself

### CODE REVIEW

File: content/community/sig/switzerland/index.md
Comment (bgruening) [inline content/community/sig/switzerland/index.md:6]: at least 697 people :)

```suggestion
Only few people are using Galaxy in Switzerland and we try to make things easier for any people in Switzerland wishing to contact other Swiss people.
```

**Review by bgruening — APPROVED** [review]

**Comment by lldelisle** [comment]
I think we need to wait for @natefoo to generate the mailing list before we merge this. I will come back to him in August.

**Comment by hrhotz** [comment]
I am actually, just trying to add some text....but keep getting errors 🤔

**Comment by hrhotz** [comment]
now it worked  -  feel free to further edit

I am happy to ping Nate as well, in order to move forward. We don't need "formal" approval of our new community Wendi is already happy with it.

**Comment by bgruening** [comment]
I tried to solve the merge conflict.

**Comment by lldelisle** [comment]
@natefoo would you mind to create the mailing list https://lists.galaxyproject.org/lists/galaxy-switzerland@lists.galaxyproject.org, please?

**Pull request merged**
